src/client/styles/index.scss

```diff
@@ -20,7 +17,6 @@ body {
 }
 
 a {
-  -webkit-text-decoration-skip: objects;
```

Comment:
lmao did I have this in there? seriously I don't even know why people starred this project! 😂